pandas/tests/io/test_pytables.py

```diff
@@ -2793,10 +2793,20 @@ def test_empty_series_frame(self):
         self._check_roundtrip(df2, tm.assert_frame_equal)
 
     def test_empty_series(self):
-        for dtype in [np.int64, np.float64, np.object, 'm8[ns]', 'M8[ns]']:
+        for dtype in [np.int64, np.float64, np.object, 'm8[ns]', 'M8[ns]',
```

Comment:
Can you use `@pytest.mark.parametrize` to parametrize this test over `dtype` instead of using a `for` loop?

Reply:
Done!